### ISSUE TITLE
equal lists don't match

### ISSUE THREAD

**Issue by francisdb** [softwaremill/diffx#16]
```scala
    List("a") should equal(List("a"))

    List("a") should matchTo(List("a"))
```

the assertion fails but the shown diff shows no differences

<img width="317" alt="image" src="https://user-images.githubusercontent.com/161305/65326114-002aa580-dbb1-11e9-92a8-e596788f8d02.png">

**Comment by ghostbuster91** [comment]
:man_facepalming: How could I forgot to test it? :) 
Anyway, fixed in 0.2.1

**Comment by francisdb** [comment]
thanks for the quick fix, now using `0.2.2` as `2.1.0` is missing on central

**Comment by ghostbuster91** [comment]
Yes 0.2.1 timeouted during the maven upload, so the whole package was corrupted, and what's surprising, it also caused the next release to fail. Anyway I hope that after switching to sbt 1.3.1 we won't see these problems anymore.
